server/queue/queue-activemq/pom.xml

```diff
@@ -38,7 +38,7 @@
             <?SORTPOM IGNORE?>
             <groupId>io.netty</groupId>
             <artifactId>netty-transport</artifactId>
-            <version>4.1.72.Final</version>
+            <version>4.1.81.Final</version>
```

Comment:
You might need to ensure Netty versions are aligned

 -> S3 driver
 -> Cassandra driver
 -> Lettuce driver